addon/components/sortable-item.js

```diff
@@ -155,6 +149,7 @@ export default class SortableItemComponent extends Component {
     cloneNode.style.zIndex = '9999';
 
     this.documentWindow.classList.add('sortable-attached');
+    this.args.modifyCloneNodeOnCreate ? this.args.modifyCloneNodeOnCreate(cloneNode) : '';
```

Comment:
freshservies uses this so that, while dragging a column and if it has 100 ticket cards in it. Clone and Dragging these 100s ticket column would be a performance intense. So modifying the clone node to remove overflowing cards.

Reply:
Copied behaviour from trello boards column move.